.env

```diff
@@ -3,3 +3,5 @@ IPTS_HOSTNAME=
 IPTS_PORT=
 IPTS_SERVICE_NAME=
 IPTS_USERNAME=
+GH_APP_ID=
+GH_PEM_PATH=
```

Comment:
**[Question, non-blocking]** Out of curiosity, how is this env file deployed to the server? I'm guessing we have a production env file that lives on the server that we just edit manually? I'm curious because `.env` is listed in the `.gitignore` file but then here it is in version control as well!

https://github.com/ccao-data/service-sqoop-iasworld/blob/8e8c5fb03215bad7994c90dc0ca143a51e1a4fae/.gitignore#L1

Reply:
That's correct! There's a prod `.env` file that I force added and manually update. Once it's populated we use `skip-corktree` to ignore further changes to the file during further git operations. The `.env` file only contains non-critical variables, anything actually important is stored in `secrets/` with proper perms and takes advantage of `docker compose` local secrets.